tests/providers/influxdb/operators/test_influxDbHook.py

```diff
@@ -0,0 +1,48 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This file name and path is wrong. This PR introduce only hook so it should not be in `/operators`
The path should be:
`tests/providers/influxdb/hooks/test_influxdb.py` (note you will also need to change the `__init__`)

also you only tested the connection. the  hook contains functions `query`/`create_organization`/`find_bucket_id_by_name` and others that have no testing. is this intentional?

Reply:
Thanks, will move it to hooks, some of the functions are simple wrappers, but I can added test two functions(write and find_bucket_id_by_name)